index.html

```diff
@@ -349,6 +344,50 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+        <dt>readonly attribute Element? pointerLockElement</dt>
+        <dd>
+          <dfn title="pointerLockElement"></dfn>
+
+          <p>While the pointer is locked, returns the result of the
+          <a href="https://w3c.github.io/webcomponents/spec/shadow/#dfn-retargeting-algorithm">retargeting algorithm</a>
+          with the <a href="https://dom.spec.whatwg.org/#context-object">context object</a> and
+          the element, which is the target for mouse events, as input, if the result and the
+          <a href="https://dom.spec.whatwg.org/#context-object">context object</a> are in the
+          same tree, otherwise returns null.</p>
+          <p>Returns null if lock is pending or if pointer is unlocked.</p>
+          <pre class='example highlight' id="example-retarget-pointerlock">
+              &lt;body&gt;
+                &lt;div id=&quot;host1&quot;&gt;
+                  &lt;shadow-root id=&quot;root1&quot;&gt;
+                    &lt;canvas id=&quot;canvas1&quot;&gt;&lt;/canvas&gt;
+                  &lt;/shadow-root&gt;
+                &lt;/div&gt;
+                &lt;div id=&quot;host2&quot;&gt;
+                  &lt;shadow-root id=&quot;root2&quot;&gt;
+                    &lt;canvas id=&quot;canvas2&quot;&gt;&lt;/canvas&gt;
+                  &lt;/shadow-root&gt;
+                &lt;/div&gt;
+              &lt;/body&gt;
+          </pre>
+          <p>Note: the example uses fictional <code>shadow-root</code> element to denote a
```

Comment:
In this section I think that all ids can be used raw, without needing the element type prefixes. E.g.
canvas#canvas1 -> canvas1
div#host1 -> host1